a1/src/main/java/ro/utcn/sd/it/a1/Controller/ConsoleController.java

```diff
@@ -0,0 +1,185 @@
+package ro.utcn.sd.it.a1.Controller;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.stereotype.Component;
+import ro.utcn.sd.it.a1.exception.QuestionNotFoundException;
+import ro.utcn.sd.it.a1.exception.UserNotFoundException;
+import ro.utcn.sd.it.a1.model.Answer;
+import ro.utcn.sd.it.a1.model.Question;
+import ro.utcn.sd.it.a1.model.Tag;
+import ro.utcn.sd.it.a1.model.User;
+import ro.utcn.sd.it.a1.service.AnswerManagementSystem;
+import ro.utcn.sd.it.a1.service.QuestionManagementSystem;
+import ro.utcn.sd.it.a1.service.TagManagementSystem;
+import ro.utcn.sd.it.a1.service.UserManagementSystem;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.sql.Timestamp;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.Optional;
+import java.util.Scanner;
+
+@Component
+@RequiredArgsConstructor
+public class ConsoleController implements CommandLineRunner {
```

Comment:
Normally I would expect that you should disable / delete this class such that the application does not wait for user input.